packages/smart-contracts/hardhat.config.ts

```diff
@@ -0,0 +1,50 @@
+import '@typechain/hardhat';
+import '@nomiclabs/hardhat-waffle';
+import '@nomiclabs/hardhat-ganache';
+import '@nomiclabs/hardhat-etherscan';
+import '@nomiclabs/hardhat-ethers';
+import { task } from 'hardhat/config';
+import { config } from 'dotenv';
+import deployRequest from './scripts/1_deploy-request-storage';
+import deployPayment from './scripts/2_deploy-main-payments';
+import deployConversion from './scripts/3_deploy_chainlink_contract';
+
+config();
+
+export default {
+  solidity: '0.5.17',
+  paths: {
+    sources: 'src/contracts',
+    tests: 'test/contracts',
+    artifacts: 'build',
+  },
+  networks: {
```

Comment:
Yes. What's wrong ?

Reply:
I don't really see the point, since the WEB3 URL is configured by env var...but nevermind. it was already like this with Truffle, there must be a reason I don't grasp